src/components/LeftSidebar/ConversationsList/ConversationsList.vue

```diff
@@ -75,13 +75,25 @@ export default {
 
 	mounted() {
 		EventBus.$on('routeChange', this.onRouteChange)
+		EventBus.$on('newMessagePosted', this.onMessagePosted)
 	},
 
 	beforeDestroy() {
 		EventBus.$off('routeChange', this.onRouteChange)
+		EventBus.$off('newMessagePosted', this.onMessagePosted)
 	},
 
 	methods: {
+		onMessagePosted({ token }) {
+			const conversation = document.getElementById(`conversation_${token}`)
+			this.$nextTick(() => {
+				conversation.scrollIntoView({
+					behavior: 'smooth',
+					block: 'center',
```

Comment:
I find the center thing always weird. I think the top makes much more sense?

Reply:
yeah, I'll change to "start" as I also observed some occurrences where it stopped in the middle of a row.